dev-python/dukpy/dukpy-0.3.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="JavaScript in Python"
+DESCRIPTION="dukpy is a JavaScript runtime environment for Python (tested with python 2.7 and >= 3.4) using the duktape embeddable JavaScript engine. With dukpy, you can run JavaScript in Python."
+HOMEPAGE="https://github.com/kovidgoyal/dukpyt"
+COPYRIGHT="Duktape authors"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/kovidgoyal/dukpy/archive/v0.3.tar.gz"
+CHECKSUM_SHA256="cc78c39ebba51f381c623b164cfb7dcf3caddf515fe7094bc53b7eca5d4e435e"
+
+
+ARCHITECTURES="!x86_gcc2 x86_64"
+
+PROVIDES="
+	$portName = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	cmd:gcc
```

Comment:
cmd: gcc shouldn't be needed here, it should be moved to BUILD_PREREQUIRES

Reply:
like now?

Reply:
or like this?

Reply:
nope